.vale/styles/AsciiDoc/ValidCodeBlocks.tengo

```diff
@@ -0,0 +1,31 @@
+/*
+    Tengo Language
+    $ tengo ValidCodeBlocks.tengo <asciidoc_file_to_validate>
+*/
+
+fmt := import("fmt")
+os := import("os")
+text := import("text")
+
+input := os.args()
+scope := os.read_file(input[2])
+
+matches := []
+codeblock_delim_regex := "-{4,}"
+source_block_regex := "\\[(source,|,)[a-zA-Z0-9]+\\]"
```

Comment:
@aireilly - nit - fix is to add "subs" as an option in this regex 

I added a literal codeblock using Avani's VS Code snippets and this threw an error as it uses the subs attribute:

`[subs="+quotes",options="nowrap",role="white-space-pre"]
----
cli output
----
`

Also, callouts in codeblocks uses subs according to docs guidelines:

[subs=-callouts]
....
----
code example 1 <1>
code example 2 <2>
----
<1> A note about the first example value.
<2> A note about the second example value.
....

Reply:
Updated. PTAL.